src/app/project-overview/components/project-overview/project-overview.component.ts

```diff
@@ -89,6 +91,9 @@ export class ProjectOverviewComponent implements OnInit, OnDestroy {
 
     this.routeService.updateActivatedRoute(this.activatedRoute);
 
+    this.isManaged = ConfigManager.getIsManaged();
```

Comment:
we need to check if the config manager was initialized first compare `checkIfManagedVersion`